### ISSUE TITLE
Previously saved Page content lost

### ISSUE THREAD

**Issue by maddisondesigns** [WordPress/gutenberg#3448]
## Issue Overview
I started editing a previously saved draft page. This page only consisted of a Page Title, 1 Paragraph Block with text and 1 Image block with an image.

After clicking the image, I wanted to change it to Wide, using the toolbar options. I noticed that the Wide toolbar options weren't available and realised it was due to changing themes.

Opening the Appearance > Themes page in another tab, I switched to another theme, went back to my page and refreshed. After clicking the image and checking the toolbar again, the Wide option still wasn't available (which meant I selected the wrong them, as it obviously didn't have the theme support added).

I want back to the Themes page in my other tab, and selected the Gutenberg theme.

I then went back to my previous tab, refreshed again, and all my page content disappeared. Not only that, when I go back to my List of Pages, my page that I'd previously saved was completely missing. Instead, Now I have two untitled drafts, both of which have no content. The page that I'd previously saved, no longer exists.

![gutenbergissue-lostcontent](https://user-images.githubusercontent.com/2078245/32706451-944e6bf2-c871-11e7-978c-510abeb029f0.png)

Both untitled drafts in the above image, have no content, as shown here

![gutenbergissue-lostcontent2](https://user-images.githubusercontent.com/2078245/32706488-c92098dc-c871-11e7-9865-ffa329efe98e.png)

If I check my database for my previous content, I can see that my previous page has now mysteriously been moved to a revision. I can also see that the post_parent for those revisions, is one of the untitled drafts mentioned above. So, not only has it moved my previous content to a revision, it's removed all the content from the page including the title.

![gutenbergissue-lostcontent3](https://user-images.githubusercontent.com/2078245/32706508-ed8ba338-c871-11e7-8ee1-82690d91e5a3.png)

## Steps to Reproduce (for bugs)

1. Open saved Page (I opened a page which as in Draft)
2. Change Themes (using another browser tab)
3. Refresh Editor screen in previous tab

## Expected Behavior
Refreshing the Editor screen should simply reload the content for that Page/Post

## Current Behavior
Lost all my saved content

## Possible Solution
Don't lose content

_Gutenberg 1.6.1
Firefox 56.0.2 (64-bit)
macOS Sierra 10.12.1_

**Comment by danielbachhuber** [comment]
@maddisondesigns Any ideas why this happened technically? Can you reproduce it with a stock WordPress install, no extra plugins active?

**Comment by maddisondesigns** [comment]
@danielbachhuber No idea why this happened. I've never had anything like this happen prior to Gutenberg, in all the years I've used WP

**Comment by danielbachhuber** [comment]
@maddisondesigns Yeah, I agree that it's a bit disturbing. Can you reproduce the issue?
